kuma/api/v1/views.py

```diff
@@ -33,10 +34,48 @@ def doc(request, locale, slug):
     return JsonResponse(document_api_data(document))
 
 
-def document_api_data(document):
+def document_api_data(document, ensure_contributors=False):
+    while document.is_redirect:
+        redirect_document = document.get_redirect_document(id_only=False)
+        if redirect_document:
+            document = redirect_document
```

Comment:
Would you mind putting a `continue` here? Given how long the else clause is, I was not sure whether there was any more code in the while loop after the else. It would be simpler to see an explicit continue.